File: leetcode3/최원준/3813. Vowel-Consonant Score.py
class Solution:
    def vowelConsonantScore(self, s: str) -> int:
        vc = [0,0]
        for c in s:
            if c in ('a','e','i','o','u'):
                vc[0]+=1
            elif c.isalpha():
                vc[1]+=1
        return floor(vc[0]/vc[1]) if vc[1]>0 else 0

File: leetcode3/황은지/3813. Vowel-Consonant Score.js
/**
 * @param {string} s
 * @return {number}
 */
var vowelConsonantScore = function (s) {
  let v = 0;
  let c = 0;
  for (const char of s) {
    if (
      char === "a" ||
      char === "e" ||
      char === "i" ||
      char === "o" ||
      char === "u"
    )
      v++;
    else if (char !== " " && isNaN(char)) c++;
  }

  return c > 0 ? Math.floor(v / c) : 0;
};
